samples/app-complete-auth/nodejs/appPackage/manifest.json

```diff
@@ -140,7 +145,7 @@
   "accentColor": "#60A18E",
   "permissions": [ "identity", "messageTeamMembers" ],
   "validDomains": [
-    "<<Domain-name>>",
+    "<<base-url>>",
```

Comment:
What will be the base-url value?
Did we mention in the Readme file?

Reply:
Yes it is mentioned.